### ISSUE TITLE
function get_session/2 undefined

### ISSUE THREAD

**Issue by StefanScott** [gutschilla/phoenix-skeleton#1]
```
== Compilation error on file web/views/auth_view.ex ==
** (CompileError) web/templates/auth/index.html.eex:1: function get_session/2 undefined
    (stdlib) lists.erl:1336: :lists.foreach/2
    (stdlib) erl_eval.erl:657: :erl_eval.do_apply/6
```

**Comment by gutschilla** [comment]
Ahh, thanks for that. I am currently merging all the stuff to comply with Phoenix 0.9.0 ...

**Comment by gutschilla** [comment]
I have to port all that authentication stuff to current version of Ecto (many-to-many-assocs finally) and Postgrex.See /next2 for state of work.

**Comment by gutschilla** [comment]
It's fixed in 0.10.1
